apps/frontend/src/components/global/ExportHTMLModal.vue

```diff
@@ -247,14 +462,91 @@ export default class ExportHTMLModal extends Vue {
         StatusCountModule.countOf(this.filter, 'PassingTestsFailedControl') +
         StatusCountModule.countOf(this.filter, 'FailedTests')
     };
+    this.outputData.severity = {
+      none: SeverityCountModule.none(this.filter),
+      low: SeverityCountModule.low(this.filter),
+      medium: SeverityCountModule.medium(this.filter),
+      high: SeverityCountModule.high(this.filter),
+      critical: SeverityCountModule.critical(this.filter)
+    };
     this.outputData.files = [];
     this.outputData.controlSets = [];
   }
 
+  // Replace all found illegal characters in string with compliant string equivalent
+  replaceIllegalChar(text: string): string {
+    for (const illegalChar in ILLEGAL_CHAR_SET) {
+      text = text.replace(
+        new RegExp(`${illegalChar}`, 'g'),
+        ILLEGAL_CHAR_SET[illegalChar as keyof typeof ILLEGAL_CHAR_SET]
+      );
+    }
+    return text;
+  }
+
+  // Sets attributes for each specific control
   addDetails(
     control: ContextualizedControl,
     controlLevels: ContextualizedControl[]
-  ): ContextualizedControl & {details: Detail[]} {
+  ): ContextualizedControl & {details: Detail[]} & {controlID: string} & {
+    controlStatus: ControlStatus;
+  } & {controlSeverity: ControlSeverity} & {controlTags: string[]} {
+    // Check status of individual control to assign corresponding icon
+    let statusColor;
+    switch (control.root.hdf.status) {
```

Comment:
file an issue to come back and use the enum/better string typing that's currently in the checklistviewer pr as opposed to the string typing that we're doing here

applies to both status and severity.  that pr also includes having better and consistent aesthetic presentation of all of the things so you can get it from an authoritative source.

Reply:
Done.